src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceScopes.java

```diff
@@ -269,8 +269,9 @@ public Object readObject(String in) throws Exception {
     }
 
     public int[] getSelectedIndices() {
+        // violation below, 'Distance between .* declaration and its first usage is 2.'
         int[] sel = new int[5];
-        String model = "";
```

Comment:
Remove new line and use ` // violation 'Distance .* is 2.'` like you did below.

It best to avoid adding new lines in existing inputs, unless it is specifically part of the issue.

Reply:
@rnveach done